<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolTip</class>
 <widget class="QDialog" name="ToolTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>360</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>思源黑体</family>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>ToolTip</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../main/client/resources.qrc">
    <normaloff>:/assets/logo128.png</normaloff>:/assets/logo128.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	border-radius: 20px;
	background: transparent;
	border: 1px solid rgba(255, 255, 255, 51);
}
QWidget {
	font-family: &quot;思源黑体&quot;;
	color: white;
	background: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#centralwidget{
	border-radius: 20px;
	background: rgba(61, 71, 112, 1);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>51</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_title" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget #widget_title{
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(78, 228, 224, 0),
        stop:0.1919 rgba(77, 153, 245, 64),
        stop:1 rgba(78, 218, 227, 0));
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>14</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_title_icon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget {
    background: transparent;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
           <property name="pixmap">
            <pixmap resource="../ui.qrc">:/assets/png/tooltip.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_title_text">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>思源黑体</family>
             <pointsize>21</pointsize>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget {
    background: transparent;
}</string>
           </property>
           <property name="text">
            <string>提示</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_message">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>400</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>思源黑体</family>
          <pointsize>24</pointsize>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>提示信息</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,1">
        <property name="spacing">
         <number>99</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_accept">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>60</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>180</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(73, 126, 229, 1),
        stop:1 rgba(78, 218, 222, 1));</string>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reject">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(250, 77, 100, 1),
        stop:1 rgba(240, 112, 79, 1));</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../main/client/resources.qrc"/>
  <include location="../ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_accept</sender>
   <signal>clicked()</signal>
   <receiver>ToolTip</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reject</sender>
   <signal>clicked()</signal>
   <receiver>ToolTip</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
